packages/mcp/tsconfig.json

```diff
@@ -5,14 +5,22 @@
     "outDir": "./lib",
     "rootDir": "./src",
     "strictNullChecks": true,
-    "plugins": [{ "transform": "@salesforce/core/lib/messageTransformer", "import": "messageTransformer" }],
```

Comment:
not needed, this message transformer plugin is only used in SDR and maybe other libs to inline messages strings, we don't use sfdx-core's messages framework in the mcp:
https://github.com/forcedotcom/source-deploy-retrieve/blob/7666f89429cbb5c9c512e583339165d2d66214cd/tsconfig.json#L10